src/main/java/ch/uzh/ifi/hase/soprafs23/controller/GroupController.java

```diff
@@ -26,22 +26,22 @@ public class GroupController {
     }
 
 
-//    @PostMapping("/groups")
-//    @ResponseStatus(HttpStatus.CREATED)
-//    @ResponseBody
-//    public GroupGetDTO createGroup(@RequestBody GroupPostDTO groupPostDTO, @RequestParam Long hostId) {
-//        // convert API user to internal representation
-//        GroupGetDTO groupInput = DTOMapper.INSTANCE.convertGroupPostDTOtoEntity(groupPostDTO);
-//
-//        // get the user creating the group
-//        User host = userService.getUserById(hostId);
-//
-//        // create group
-//        GroupGetDTO createdGroup = groupService.createGroup(groupInput, host);
-//
-//        // convert internal representation of user back to API
-//        GroupGetDTO groupGetDTO = DTOMapper.INSTANCE.convertEntityToGroupGetDTO(createdGroup);
-//
-//        return groupGetDTO;
-//    }
+    @PostMapping("/groups")
+    @ResponseStatus(HttpStatus.CREATED)
+    @ResponseBody
+    public GroupGetDTO createGroup(@RequestBody GroupPostDTO groupPostDTO, @RequestParam Long hostId) {
```

Comment:
Add the authenticity check
-> check the X-Token header
      -> is there a user with that token?
      -> is the id of the user with that token the same as the hostId?